Makefile

```diff
@@ -11,6 +11,19 @@ $(MBEDTLS_PATH)/framework/exported.make not found.
 Run `git submodule update --init` to fetch the submodule contents.
 This is a fatal error
         endef
+        ifeq (,$(wildcard .git))
+          define error_message
+$(MBEDTLS_PATH)/.git/ not found.
+Please ensure you have downloaded a release version of MbedTLS from Github.
```

Comment:
In most of our docs we have Mbed TLS (with a space), might be good to keep it consistent?